tests/test_cve_api.py

```diff
@@ -0,0 +1,36 @@
+import json
+from pathlib import Path
+
+import pytest
+
+from cvelib.cve_api import CveApi, CveRecord, CveRecordValidationError
+
+
+def test_full_schema_validation():
+    with open(Path(__file__).parent / "data/CVEv5_advanced-example.json") as record_file:
+        cve_json = json.load(record_file)
+    try:
+        CveRecord.validate(cve_json, CveRecord.Schemas.V5_SCHEMA)
+    except CveRecordValidationError as exc:
+        pytest.fail(f"{exc}: {exc.errors}")
```

Comment:
Any reason to use pytest.fail instead of just letting the error get raised normally? I've seen this in some other code but don't really understand what the advantage / difference is. Prettier formatting?